test/test_cli_integration.py

```diff
@@ -30,96 +31,118 @@ def setup_cli_params(self, env_vars, *params):
+            self.setup_cli_params({})
+            output = self.invoke_cli(self.cli_auth_params,
+                                     ['init'], exit_code=1)
 
-        self.assertRegex(output, "You decided not to trust the certificate")
-        assert not os.path.isfile("/root/.conjurrc")
+            self.assertRegex(output, "You decided not to trust the certificate")
+            assert not os.path.isfile(self.environment.path_provider.conjurrc_path)
 
     '''
     Validates that when the user adds the force flag, 
     no confirmation is required 
     '''
+
     @integration_test
     # The additional side effects here ('somesideffect') would prompt the CLI to
     # request for confirmation which would fail the test
     @patch('builtins.input', side_effect=['yes', 'somesideeffect', 'somesideeffect'])
     def test_https_conjurrc_user_forces_overwrite_does_not_request_confirmation(self, mock_input):
         self.setup_cli_params({})
-        output = invoke_cli(self, self.cli_auth_params,
-            ['init', '--url', 'https://conjur-https', '--account', 'dev', '--force'], exit_code=0)
+        output = self.invoke_cli(self.cli_auth_params,
+                                 ['init', '--url', self.client_params.hostname, '--account', 'dev', '--force'],
+                                 exit_code=0)
 
         assert "Not overwriting" not in output
 
     @integration_test
     def test_https_cli_fails_if_cert_is_bad(self):
-        shutil.copy('./test/test_config/bad_cert_conjurrc', '/root/.conjurrc')
+        # write bad_file
+        with open(self.environment.path_provider.conjurrc_path, 'w') as bad:
+            bad.write("---\n")
+            bad.write("account: dev\n")
+            bad.write(f"appliance_url: {self.client_params.hostname}\n")
+            bad.write(f"cert_file: {self.environment.path_provider.test_bad_cert_conjurrc_file_path}\n")
+            bad.write("plugins: []")
         self.setup_cli_params({})
         self.print_instead_of_raise_error(requests.exceptions.SSLError, "SSLError", exit_code=1)
 
     @integration_test
     def test_https_cli_fails_if_cert_is_not_provided(self):
-        shutil.copy('./test/test_config/no_cert_conjurrc', '/root/.conjurrc')
+        #write bad_file
+        with open(self.environment.path_provider.conjurrc_path,'w') as bad:
+            bad.write("---\n")
+            bad.write("account: dev\n")
+            bad.write(f"appliance_url: {self.client_params.hostname}\n")
+            bad.write("cert_file: \n")
+            bad.write("plugins: []")
         self.setup_cli_params({})
         self.print_instead_of_raise_error(requests.exceptions.SSLError, "SSLError", exit_code=1)
 
+
 # Not coverage tested since integration tests doesn't run in
 # the same build step
-class CliIntegrationTest(unittest.TestCase): # pragma: no cover
-    shutil.copy('./test/test_config/conjurrc', '/root/.conjurrc')
+class CliIntegrationTest(IntegratioTestCaseBase):  # pragma: no cover
```

Comment:
`IntegratioTestCaseBase` -> `IntegrationTestCaseBase`

In other words, missing an N